src/manifolds/KendallsPreShapeSpace.jl

```diff
@@ -0,0 +1,149 @@
+
+@doc raw"""
+    KendallsPreShapeSpace{n,k} <: AbstractSphere{ℝ}
+
+Kendall's pre-shape space of ``k`` landmarks in ``ℝ^n`` represented by n×k matrices.
+In each row the sum of elements of a matrix is equal to 0. The Frobenius norm of the matrix
+is equal to 1 [^Kendall1984][^Kendall1989].
+
+The space can be interpreted as tuples of ``k`` points in ``ℝ^n`` up to simultaneous
+translation and scaling of all points, so this can be thought of as a quotient manifold.
+
+# Constructor 
+
+    KendallsPreShapeSpace(n::Int, k::Int)
+
+# References
+
+[^Kendall1989]:
+    > D. G. Kendall, “A Survey of the Statistical Theory of Shape,” Statist. Sci., vol. 4,
+    > no. 2, pp. 87–99, May 1989, doi: 10.1214/ss/1177012582.
```

Comment:
Could you turn the DOIs into links? I went through all of the doc strings to make them links, that renders nicer in the docs and makes the references more accessible..

Reply:
Sure, no problem.